src/java.rmi/share/classes/sun/rmi/runtime/RuntimeUtil.java

```diff
@@ -36,10 +36,45 @@
  * There is a single instance of this class, which can be obtained
  * with a getInstance() call.
  *
+ * This class also contains a couple static methods for creating
+ * threads. The methods allow the choice of the Runnable for the
+ * new thread to execute, the name of the new thread (which will
+ * be prefixed with "RMI "), and whether or not it will be a daemon
+ * thread.
+ *
+ * The new thread may be created in the system thread group (the root
+ * of the thread group tree) or an internally created non-system
+ * thread group (the "user" thread group).
+ *
+ * The new thread will have the system class loader as its initial
+ * context class loader (that is, its context class loader will NOT be
+ * inherited from the current thread).
+ *
  * @author      Peter Jones
  **/
 public final class RuntimeUtil {
 
+    /**
+     * Cached reference to the system (root) thread group.
+     */
+    static final ThreadGroup systemThreadGroup;
```

Comment:
Can this be private?  Also userThreadGroup below?

Reply:
Good point. They weren't private in the original, which I just copied over, but I guess we're being a bit more meticulous about making things private these days.